servant-mock/test/Servant/MockSpec.hs

```diff
@@ -40,8 +38,11 @@ data TestHeader
   | ArbitraryHeader
   deriving (Show)
 
-instance ToByteString TestHeader where
-  builder = fromString . show
+instance ToHttpApiData TestHeader where
+  toHeader = toHeader . show
+  toUrlPiece _ = error "ToHttpApiData.toUrlPiece not implemented for TestHeader"
```

Comment:
I prefer implementing because (a) it's shorter and less attention grabbing, and (b) in my opinion tests should sort of set an example for how to do things.